### PR TITLE
chore: add command to serve docs locally

### PR DESCRIPTION
**Pull request by mobot11**
### Description
This command allows devs to serve a 'production' build of the docs site locally for debugging purposes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,6 +120,7 @@
     "react-test-renderer": "^16.6.3",
     "resolve": "1.8.1",
     "sass-loader": "7.1.0",
+    "serve": "^10.1.2",
```
Comment (greg-a-smith) [inline package.json]: Does this have a specified port or maybe more specifically, is the port number configurable?

Comment (mobot11) [reply package.json]: It serves from port 5000 by default. You can configure it to serve from any port with the -l flag. running `serve -h` gives all of the possible configuration options.

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
     "build-doc": "node scripts/build.js",
+    "build-doc:local": "BUILD_ENV=development npm run build-doc && serve -s build",
```
Comment (bcullman) [inline package.json]: can we just make this `serve` as in `npm run serve`

**Comment by bcullman** [comment]
master is not merged into this yet, so running serve locally does uncover #406 

After I locally merged master, I was able to see docs locally again. This is a good feature.

**Comment by mobot11** [comment]
@greg-a-smith I'm guessing this is because this branch isn't up to date with master. I just rebased it can you try again?

**Pull request merged**